### DISCUSSION TITLE
Set Variable With Null Value

### DISCUSSION THREAD

**Discussion by harinugrohotranstrack** [postmanlabs/postman-flows#348] [Q&A] [answered]
Hello, i have 2 cases one of which requires a variable with null value is there any solution for this? Thankyou
<img width="847" alt="Screen Shot 2022-07-12 at 08 34 42" src="https://user-images.githubusercontent.com/108118516/178389555-9ebd1b78-df00-4fde-ba5d-b6020683ce74.png">

**Comment by shrey-bhagat** [comment, marked as answer]
@harinugrohotranstrack you can still use the variable with _null_ value despite the error shown above. We'll fix this issue ASAP
